agentops/client.py

```diff
@@ -119,7 +119,9 @@ def _check_for_partner_frameworks(self, instrument_llm_calls, auto_start_session
                 if framework == 'autogen':
                     try:
                         import autogen
-                        autogen.runtime_logging.start(logger_type="agentops")
+                        from .partners.autogen_logger import AutogenLogger  # TODO: move?
+                        # TODO: can we pass it like this without it being a singleton?
```

Comment:
We can remove these TODOs

Reply:
i did not see these:

1. we should only import if needed, so i would say lets keep the import in the conditional
2. it works, so i guess so but honestly idk why and havent looked into it